<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>371</width>
      <height>401</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>241</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>操作区</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>361</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="pushButtonPlay">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>51</width>
        <height>45</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>开</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonStop">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>51</width>
        <height>45</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>暂</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonContinue">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>51</width>
        <height>45</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>继</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRestart">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>51</width>
        <height>45</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>重</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSave">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>0</y>
        <width>51</width>
        <height>45</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>保</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLoad">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>51</width>
        <height>45</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>载</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonQuit">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>0</y>
        <width>51</width>
        <height>45</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>退</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>障碍物</string>
     </property>
     <widget class="QPushButton" name="pushButtonLine">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>单个设置</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>蛇速度</string>
     </property>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>28</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="maximum">
       <number>49</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>娱乐</string>
     </property>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>24</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>食物移动</string>
      </property>
     </widget>
    </widget>
    <widget class="QLCDNumber" name="lcdNumberTime">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>时间</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>分数</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumberScore">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>90</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="Frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="edgeLabel">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>410</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlay">
    <property name="title">
     <string>游戏</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionStop"/>
    <addaction name="actionContinue"/>
    <addaction name="actionRestart"/>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuPlay"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPlay">
   <property name="text">
    <string>开始游戏</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>暂停游戏</string>
   </property>
  </action>
  <action name="actionContinue">
   <property name="text">
    <string>继续游戏</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>重新开始</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存游戏</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>载入游戏</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出游戏</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>游戏帮助</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
